screens/index.js

```diff
@@ -26,6 +28,20 @@ const getBreakPoint = (width) => {
 };
 
 const MenuList = () => {
```

Comment:
Let's keep naming consistent. Instead of calling the component `MenuList` let's call it `Index` since this component contained the Index page.